worker/remoterelations/relationunitsworker.go

```diff
@@ -69,7 +69,9 @@ func (w *relationUnitsWorker) Kill() {
 
 // Wait is defined on worker.Worker
 func (w *relationUnitsWorker) Wait() error {
-	return w.catacomb.Wait()
+	err := w.catacomb.Wait()
+	logger.Errorf("error in relation units worker for %v: %v", w.relationTag.Id(), err)
```

Comment:
Looks like this will be emitted even when error is nil...

Reply:
yup, silly me, fixed